aries-site/src/examples/components/toast-notifications/ToastNotification.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import { StatusGoodSmall } from 'grommet-icons/icons/StatusGoodSmall';
+import { FormClose } from 'grommet-icons/icons/FormClose';
+
+import { Box, Button, Layer, Text } from 'grommet';
+
+const Notification = ({ toast, title, message, onClose }) => {
```

Comment:
Keep name of export and file name consistent? Style/convention preference.

```suggestion
const ToastNotification = ({ toast, title, message, onClose }) => {
```

Reply:
Or alternatively this file could be called `Notification.js`. Presumably this file becomes obsolete once Notification is available by Grommet.

Reply:
moved to be in the notifications folder in `src/examples/components/notifications` and renamed the file to be `Notification.js`